x-pack/plugin/sql/qa/src/main/resources/select.csv-spec

```diff
@@ -1,3 +1,27 @@
+//
+// SELECT with = and !=
+equalsAndNotEqualsSelectClause
+SELECT 4 = 4, NOT 4 = 4, 4 != 4, NOT 4 != 4, 3 = 4, NOT 3 = 4, 3 != 4, NOT 3 != 4, 1 = null, NOT null = 1, 1 != null, NOT 1 != null;
+
+  4 = 4 |  NOT 4 = 4 |  4 != 4 |  NOT 4 != 4 |  3 = 4 |  NOT 3 = 4 |  3 != 4 |  NOT 3 != 4 |  1 = null |  NOT null = 1 |  1 != null |  NOT 1 != null
+--------+------------+---------+-------------+--------+------------+---------+-------------+-----------+---------------+------------+---------------
+true    |false       | false   | true        | false  | true       | true    | false       | null      | null          | null       | null
+;
+      
+      
+equalsAndNotEqualsSelectClauseWithTableColumns
+SELECT languages = 2, NOT languages = 2, languages != 2, NOT languages != 2, languages != null, NOT null != languages
+FROM "test_emp" WHERE emp_no IN(10018, 10019, 10020) ORDER BY emp_no ;
+
+2
```

Comment:
Are these 3 lines accepted by the test class? I'm assuming you added these to show which `languages` values you are testing against. If so, why not showing the `languages` column as well?

Reply:
no this was a leftover, I'll remove them but I will merge #35239 first which fixes and enables existing tests.